components/experimental/src/dimension/provider/currency/fractions_ule.rs

```diff
@@ -64,10 +63,10 @@ impl AsULE for FractionInfo {
             } else {
                 Some(cash_digits)
             },
-            cash_rounding: if cash_rounding == NONE_MARKER {
-                None
-            } else {
-                Some(cash_rounding)
+            cash_rounding: match cash_rounding {
+                14 => Some(50),
```

Comment:
Observation: this is a rounding increment. It might make sense to encode this as an enumeration. What you have here is kind of an enumeration, with 0-13 as identities, 14 = 50, and 15 = none. Increments of 3, 4, 6, and other numbers not factors of 10 or 100 seem unlikely to be used, so we could use those values. What you have here is perfectly fine since it can be extended in the future.